trade-finance-manager-ui/templates/utilisation-reports/utilisation-report-reconciliation-for-bank.njk

```diff
@@ -0,0 +1,79 @@
+{% from "govuk/components/tabs/macro.njk" import govukTabs %}
+
+{% extends "index.njk" %}
+
+{% set premiumPaymentsHtml %}
+  <div></div> {# TODO: FN-1615 #}
+{% endset %}
+
+{% set keyingSheetHtml %}
+  <div></div> {# TODO: FN-1722 #}
+{% endset %}
+
+{% set paymentDetailsHtml %}
+  <div></div> {# TODO: No tickets created yet #}
+{% endset %}
+
+{% set utilisationHtml %}
+  <div></div> {# TODO: No tickets created yet #}
```

Comment:
Similar to my other comment with regards to TODOs, but even more so let's please not have any TODOs without ticket numbers.
I can see the ACs do say we should all four tabs but also I'm questioning that slightly since Christian mentioned we may not end up doing the utilisation tab. It would make more sense for tickets populating tabs to add their own tabs past the first one. @ChristianMcc should/could we descope any of these tabs from this ticket?

Reply:
For now, I'll just remove the TODOs. In terms of keeping the four tabs, I don't see much downside in meeting the ACs here and removing specific tabs later if we decide not to keep them, but I'll do whatever @ChristianMcc says :)

Reply:
I reckon remove TODOs, leave in tabs (because it's probably easier to raise a new PR than to go through changing ACs and while I'm unsure if the last tab will exist I'm not confident enough to base a decision on that just yet :D